include/swift/SILOptimizer/Utils/CanonicalizeOSSALifetime.h

```diff
@@ -409,6 +421,10 @@ class CanonicalizeOSSALifetime final {
   void recordConsumingUser(SILInstruction *user) {
     consumingBlocks.insert(user->getParent());
   }
+  void recordUnreachableLifetimeEnd(SILInstruction *user) {
```

Comment:
At the definition of `consumingBlocks` and `recordConsumingUser` we should comment that these are either the points where the original value was consumed or was available at an unreachable instruction.

Reply:
Fixed.